e2e/testcases/helm_sync_test.go

```diff
@@ -405,7 +405,10 @@ image:
 				  "value": "inline"
 				}
 			  ],
-			  "wordpressEmail": "test-user@example.com"
+			  "wordpressEmail": "test-user@example.com",
+			  "service": {
+				"type": "ClusterIP"
```

Comment:
missing a trailing comma

Reply:
This is JSON rather than Go, so I think the validation error may actually be coming from the superfluous trailing comma on line 411

Reply:
Right, it's the last item in values so shouldn't need to trailing comma. Updated.